File: api/Models/Charity.cs
namespace SuspendedStorefront.Models
{
    public class Charity
    {

        public Guid ID { get; set; }

        public Guid AdministratorID { get; set; }
        public string Name { get; set; }

        public string PictureURL { get; set; }
        public string Address1 { get; set; } = "";
        public string Address2 { get; set; } = "";
        public string City { get; set; } = "";
        public string County { get; set; } = "";
        public string PostalCode { get; set; } = "";

        public bool IsActive { get; set; }

        public int Status { get; set; }

        public Customer Administrator { get; set; }

        public IEnumerable<CharityProduct> CharityProducts {get;set;}

}
}

File: ClientApp/SuspendedStorefront/src/app/api/models/charity.ts
/* tslint:disable */
/* eslint-disable */
import { CharityProduct } from './charity-product';
import { Customer } from './customer';
export interface Charity {
  address1?: null | string;
  address2?: null | string;
  administrator?: Customer;
  administratorID?: string;
  charityProducts?: null | Array<CharityProduct>;
  city?: null | string;
  county?: null | string;
  id?: string;
  isActive?: boolean;
  name?: null | string;
  pictureURL?: null | string;
  postalCode?: null | string;
  status?: number;
}
